doc/Settings.md

```diff
@@ -0,0 +1,46 @@
+# WinGet CLI Settings
+
+You can configure WinGet by editing the `settings.json` file. The file can be open with the default json editor by running `winget settings`. If no editor is configure, notepad.exe will be used.
+
+## File Location
+
+Settings file is located in %LOCALAPPDATA%\Packages\Microsoft.DesktopAppInstaller_8wekyb3d8bbwe\LocalState\settings.json
+
+If you are using the non-packaged winget version by building it from source code the file will %LOCALAPPDATA%\Microsoft\WinGet\Settings\settings.json
+
+## Source
+
+These settings involve configuration to the WinGet source.
+
+```
+    "source": {
+        "autoUpdateIntervalInMinutes": 3
+    },
+``` 
+
+### autoUpdateIntervalInMinutes
+
+Positive integer that represents the interval in minutes on how often to update the WinGet source automatically. For no updates use 0.
```

Comment:
>Positive integer that represents the interval in minutes on how often to update the WinGet source automatically [](start = 0, length = 111)

Positive integer that represents the interval in minutes of how often to automatically check for updates to a WinGet source. The check for updates only happens when a source is used, and if no update is available the interval will be reset.